Apps/WebClient/src/Server/Models/UserNote.cs

```diff
@@ -96,7 +96,7 @@ public static UserNote CreateFromDbModel(Database.Models.Note model, ICryptoDele
             {
                 Id = model.Id,
                 HdId = model.HdId,
-                JournalDateTime = model.JournalDateTime,
+                JournalDateTime = DateTime.SpecifyKind(model.JournalDateTime, DateTimeKind.Unspecified),
```

Comment:
Is this effective because after creation it is fetched back to the frontend?

Reply:
When it comes from the front end the kind comes as UTC, while when it loads from the database it comes as Unspecified, when its in UTC the frontend converts it to local time and messes up with the date.